inspirehep/modules/theme/templates/inspirehep_theme/format/record/Holding_Pen_HTML_detailed.tpl

```diff
@@ -20,7 +20,7 @@
 {% block details_page %}
 
   {% from "inspirehep_theme/format/record/Holding_Pen_HTML_detailed_macros.tpl" import record_arxiv_pdf, add_delimiter, get_abstract with context %}
-  {% from "inspirehep_theme/format/record/Inspire_HTML_detailed_macros.tpl" import record_publication_info, record_doi, record_keywords with context %}
+  {% from "inspirehep_theme/format/record/Inspire_HTML_detailed_macros.tpl" import record_publication_info, record_oi, record_keywords with context %}
```

Comment:
Hmm... strange though that I haven't seen any error due to this :/, what should this be breaking?